javascript/javascript2/week2/event-application1.js

```diff
@@ -0,0 +1,15 @@
+function myFunction(numdays) {
+    const alldays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+    d = new Date()
```

Comment:
Variable should have type, I mean is it const, or let? define appropriate type for new date. 
Variable name should be meaningful. Using alphabets like 'd' is not coding standard, so give meaningful and self descriptive name. 
go through the below resource 
https://www.youtube.com/watch?v=NibsUd_InPU